test/integration/driver_type_test.go

```diff
@@ -0,0 +1,59 @@
+// +build integration
```

Comment:
There was no need to add integraiton test to test this,

please remove this and add a unit test instead.

Reply:
yup, thanks for the suggestion :)